static/docs/use-cases/shared-storage-on-nfs.md

```diff
@@ -0,0 +1,254 @@
+# Shared Storage on NFS
+
+Data is increasing exponentially in terms of size and information. In the modern
+software development environment, teams are working together on same dataset to
+get the results. It became necessary that data is accessible to everyone and
+every team member has a same updated dataset. For this example, we will be using
+NFS (Network File System) for storing and sharing files on the network.
+
+With DVC, we need not to copy the dataset on our local machine everytime when
+new data is added to dataset. We can set the `cache directory` on NFS server.
+The cached data will be present in the NFS server, which in turn will be fast to
+access and process requests faster.
+
+> With large data files it is better to set the cache directory to NFS. Not only
+> just it will cache the data faster but also version the data. Thus you would
+> avoid copying large files from you NFS server to the machine.
+
+Lets configure NFS server and client to see how DVC can help to make the
+workflow faster.
+
+<details>
+
+# Setting up the NFS server
```

Comment:
Is there a link we can share instead? Setting up NFS server/client is not DVC-specific and I would avoid including this into the doc.